tests/test_reference_velocity_verlet_integrator.py

```diff
@@ -23,20 +21,20 @@ def self_inverse(x, v):
         x_next, v_next = update_fxn(x, v)
         return x_next, -v_next
 
-    close = partial(np.allclose, atol=atol)
+    x0 = FIXED_TO_FLOAT(FLOAT_TO_FIXED(x0))
+    v0 = FIXED_TO_FLOAT(FLOAT_TO_FIXED(v0))
```

Comment:
Should capture comment from offline about the importance of this step!

(i.e. bitwise determinism applies for starting conditions where `FIXED_TO_FLOAT(FLOAT_TO_FIXED(x0)) == x0` ..., but that condition won't be true in general for all floating-point inputs `x0`, `v0`. However, arbitrary floating-point inputs can be made to satisfy condition by applying the round-trip conversion in these 2 lines.)

Reply:
Added in [f575314](https://github.com/proteneer/timemachine/pull/789/commits/f5753143c43232dcf5476b96f128d6597d88eca1)